### PR TITLE
merging 2 graphs

### PR DESCRIPTION
**Pull request by Frankkwizera**

### CODE REVIEW

File: seshdash/models.py [hunk 1]
```diff
@@ -253,7 +253,30 @@ class BoM_Data_Point(models.Model):
...
+         "AC_Load_in" : "V",		
+         "AC_Load_out" : "V",		
+         "pv_production" : "W",		
+         "main_on" : "V",		
+         "relay_state": "",		
+         "trans" : "",		
+         "genset_state" : "V",		
+         "site" : "",		
+         "AC_output_absolute" : "V",
+         "cloud_cover":"Okta",
```
Comment (jlark) [inline seshdash/models.py]: okta?

File: seshdash/views.py [hunk 2]
```diff
@@ -782,13 +782,16 @@ def graphs(request):
             data_values = []
             time_delta = time_delta_dict[time]
             time_bucket=time_bucket_dict[time]
-            SI_unit = get_measurement_unit(choice)
+            SI_units = BoM_Data_Point.SI_UNITS
+            SI_unit = SI_units.get(choice,'V')		
+           # SI_unit = get_measurement_unit(choice)
+            print SI_unit
```
Comment (jlark) [inline seshdash/views.py]: let's remove print statments before PR. you can use logger.debug() if you think it's a useful debug message

**Comment by jlark** [comment]
Can you include a screen shot of the final product?

File: seshdash/models.py (hunk 1 above)
Comment (Frankkwizera) [inline seshdash/models.py]: cloud cover

File: seshdash/views.py (hunk 2 above)
Comment (Frankkwizera) [inline seshdash/views.py]: screen

**Comment by jlark** [comment]
thanks @Frankkwizera what;s wrong with this picture? Can you sort out that they are not doing this double wave? There should only be one point per day.

File: seshdash/views.py (hunk 2 above)
Comment (Frankkwizera) [inline seshdash/views.py]: daily = one point per hour
weekly = one point per day
Monthly = one point per 5 days

**Comment by Frankkwizera** [comment]
weekly
weekly

**Comment by Frankkwizera** [comment]
monthly
Monthly

**Comment by Frankkwizera** [comment]
daily
Daily

File: seshdash/models.py (hunk 1 above)
Comment (jlark) [inline seshdash/models.py]: Haha just learning this : ) Let's use % instead.

**Comment by jlark** [comment]
@Frankkwizera I wasn't able to see the Default values set for the graph. Can you make Pv_production and cloud_cover?

If it is already set can you show me where?

**Comment by jlark** [comment]
@Frankkwizera can you merge conflicts and commit?

**Pull request merged**